docs/user-guide/features/sso/managing-users.md

```diff
@@ -0,0 +1,36 @@
+# Managing users
+
+## Onboarding Users and Groups
+
+### Add/remove users
+1. Open this file: `management/global/sso/locals.tf`
+2. Locate the users map within the local variables definition
+3. Add an entry to the users map with all the required, including the groups the user should belong to
```

Comment:
all the required data/information? everything required? It sounds a little truncated? I'm not really sure, maybe it is grammatically correct either way.

Reply:
Done!